src/multiPortConnector.js

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright 2022, Tony Atkins
+ * Copyright 2011-2020, Colin Clark
+ *
+ *  Licensed under the MIT license, see LICENSE for details.
+ */
+
+(function (fluid) {
+    "use strict";
+    var youme = fluid.registerNamespace("youme");
+
+    // Enclosing grade so that we can recreate our pool of connections.
+    fluid.defaults("youme.multiPortConnector", {
+        gradeNames: ["fluid.modelComponent"],
+
+        direction: "inputs",
+
+        model: {
+            connectionPorts: [],
+            ports: "{youme.system}.model.ports"
+
+        },
+
+        invokers: {
+            open: {
```

Comment:
You may want to replace all of these invokers and the calling machinery with a simple boolean flag in the model 
{
    open: false
}
which is relayed to all of the children, and then rely on local listeners to react to that state.

Reply:
The more I thought about this on my morning ride, the more I liked the idea, as it both improves stability, and cleanly restores control over whether the connection is started, including on startup.  I made a go at this and made the required changes to the port connector grades as well.